lib/funky/video.rb

```diff
@@ -40,6 +40,11 @@ def picture
       data[:picture]
     end
 
+    # @return [String] the name of Facebook page for the video.
+    def page_name
+      data.fetch(:from)[:name]
```

Comment:
I thought `data.fetch(:from)[:name]` was preferable slightly over `data[:from][:name]` because they are just same thing, and (when the day comes) it will raise `KeyError` instead of `NoMethodError` :) How do you think?